Library/Formula/eprover.rb

```diff
@@ -1,9 +1,9 @@
 class Eprover < Formula
   desc "Theorem prover for full first-order logic with equality"
-  homepage "https://www4.informatik.tu-muenchen.de/~schulz/E/E.html"
-  url "https://www4.in.tum.de/~schulz/WORK/E_DOWNLOAD/V_1.8/E.tgz"
-  version "1.8"
-  sha256 "636a5353046680f9c960d02d942df0a55af2e3941676df76e3356a334f6e842e"
+  homepage "http://eprover.org"
+  url "http://wwwlehre.dhbw-stuttgart.de/~sschulz/WORK/E_DOWNLOAD/V_1.9/E.tgz"
```

Comment:
This actually still seems to work with `https://` so let's keep that protocol. The rest is :+1: